app/src/test/java/com/appunite/loudius/ui/pullrequests/PullRequestsViewModelTest.kt

```diff
@@ -43,13 +44,15 @@ class PullRequestsViewModelTest {
         pullRequestRepository.setCurrentUserPullRequests { neverCompletingSuspension() }
         val viewModel = createViewModel()
 
+        mockkObject()
         assertTrue(viewModel.state.isLoading)
     }
 
     @Test
     fun `WHEN init THEN display pull requests list`() = runTest {
         val viewModel = createViewModel()
 
+        // I think it would be better to test non empty list there
```

Comment:
It tests a non-empty list, could you elaborate a little more on this?

Reply:
Oh... Ignore this comment, I thought there is `listOf()` instead of `listOf(Defaults.pullRequest())`